protos/v0/jvs_plugin_service.proto

```diff
@@ -42,3 +44,13 @@ message ValidateJustificationResponse {
   repeated string warning = 2;
   repeated string error = 3;
 }
+
+// GetUiDataRequest is to retrieve data for display. Currently, it is designed to obtain the display_name and hint. 
+message GetUiDataRequest {
+}
+
+// The GetUiDataResponse comprises the data that will be displayed. At present, it exclusively includes the display_name and hint.
+message GetUiDataResponse {
```

Comment:
Q: Should I use GetUiDataResponse or UiData here? 
background: the guidance from AIP (https://google.aip.dev/131) that UiData can be used directly.

Reply:
I defer to @yolocs on this question. I personally will use UiData to simplify the logic

Reply:
Update the response name to `UIData`